src/meltano/core/project_init_service.py

```diff
@@ -91,7 +97,9 @@ def create_files(self, add_discovery=False):
             add_discovery: Add discovery.yml file to created project
         """
         click.secho("Creating project files...", fg="blue")
-        click.echo(f"  {self.project_name}/")
+
+        if not self.project_directory.samefile(Path.cwd()):
+            click.echo(f"  {self.project_directory}/")
```

Comment:
`Path.samefile` requires the directory to exist:

```suggestion
        os.makedirs(self.project_directory, exist_ok=True)

        if not self.project_directory.samefile(Path.cwd()):
            click.echo(f"  {self.project_directory}/")
```